src/generators/net_caffe.cc

```diff
@@ -309,6 +310,36 @@ namespace dd
     //fparam->set_value(0.2); //TODO: option
   }
 
+
+
+  void NetLayersCaffe::add_lstm(caffe::NetParameter *net_param,
```

Comment:
Thanks for the addition, this is directly useful to other tasks such as OCR etc...